state.go

```diff
@@ -61,7 +61,11 @@ func (m *MessageReq) Copy() *MessageReq {
 		mm.View = m.View.Copy()
 	}
 	if m.Proposal != nil {
-		mm.Proposal = append([]byte{}, m.Proposal...)
+		mm.SetProposal(m.Proposal)
+	}
+	// FIXME: Check if Seal should be copied as well or instantiated an empty slice instead?
```

Comment:
@ferranbt Before merging this commit onto main, do you agree with copying `Seal` as well, since I wasn't sure if it should be empty slice or nil maybe.

Reply:
Yes, Seal is also copied.